src/lightly_train/_modules/teachers/dinov2/__init__.py

```diff
@@ -2,4 +2,5 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 #
 # This source code is licensed under the Apache License, Version 2.0
-# found in the LICENSE file in the root directory of this source tree.#
+# found in the LICENSE file in the root directory of this source tree.
+#
```

Comment:
Small cleanup to remove the weird trailing `#` on the last line. This is fixed by adding a blank line at the end of the header files.